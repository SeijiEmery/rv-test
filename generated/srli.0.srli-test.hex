0000000    93  d0  10  00  13  51  11  00  93  d1  11  00  13  52  12  00
0000020    93  d2  12  00  13  00  00  00  13  00  00  00  13  00  00  00
0000040    13  00  00  00  13  00  00  00  73  00  10  00                
0000054
