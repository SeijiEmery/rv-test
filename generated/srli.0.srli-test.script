load /x 0 generated/srli.0.srli-test.hex
writereg 1 1
writereg 2 0
writereg 3 31
writereg 4 64
writereg 5 62
setpc 0
run 1004
readreg 1
readreg 2
readreg 3
readreg 4
readreg 5