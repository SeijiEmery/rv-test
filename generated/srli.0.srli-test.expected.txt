R1 = 0
R2 = 0
R3 = 15
R4 = 32
R5 = 31
